tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {ansible2.2}-{xenial-conf-tests,xenial-mon-osd,xenial-cluster,centos7-mon-osd,centos7-cluster}
+envlist = {ansible2.2}-{xenial-conf-tests,xenial-mon-osd,xenial-cluster,raw-multi-journal,centos7-cluster,dmcrypt-dedicated-journal,dmcrypt-journal-collocation}
```

Comment:
can we do `journal` and `journal-collocated` ? I think that we should assume dedicated if not specified, and always specify 'collocated' when needed